ember/app/components/mapbox-map.js

```diff
@@ -7,6 +7,12 @@ import centerOfMass from 'npm:@turf/center-of-mass';
 
 mapboxgl.accessToken = 'pk.eyJ1IjoiaWhpbGwiLCJhIjoiY2plZzUwMTRzMW45NjJxb2R2Z2thOWF1YiJ9.szIAeMS4c9YTgNsJeG36gg';
```

Comment:
Access tokens for APIs generally should not be checked into GitHub repos. Typically we use environment variables or secrets.yml to obscure these. Unless it is a public access token that is ok to expose to the public (not tied to our account limits, etc.)